### PR TITLE
Restore the splatting feature of the Polyhedron demo

### PR DESCRIPTION
**Pull request by maxGimeno**
This is a fix for the Splatting feature in the Polyhedron_demo, removing the dependency to glew.

### CODE REVIEW

**Comment by lrineau** [comment]
Will fix #166.

File: Polyhedron/demo/Polyhedron/Scene.cpp [hunk 1]
```diff
@@ -245,6 +268,8 @@ Scene::drawWithNames(Viewer_interface* viewer)
 void 
 Scene::draw_aux(bool with_names, Viewer_interface* viewer)
 {
+    if(!ms_splatting->viewer_is_set);
+        ms_splatting->setViewer(viewer);
```
Comment (lrineau) [inline Polyhedron/demo/Polyhedron/Scene.cpp]: There is a bug, there. Hopefully, [there was a compiler smart enough to warn about it][1]:
```
/home/cgal_tester/build/src/cmake/platforms/ArchLinux-clang-Release/test/Polyhedron_Demo/Scene.cpp:271:37: warning: if statement has empty body [-Wempty-body]
    if(!ms_splatting->viewer_is_set);
                                    ^
```
[1]: https://cgal.geometryfactory.com/CGAL/Members/testsuite/CGAL-4.7-Ic-134/Polyhedron_Demo/TestReport_lrineau_ArchLinux-clang-Release.gz

**Comment by lrineau** [comment]
TODO: [fix the bug pointed out][1] in `Scene.cpp`

[1]: https://github.com/CGAL/cgal/pull/306#discussion-diff-38633460

**Comment by lrineau** [comment]
TODO: fix the bug pointed out at https://github.com/CGAL/cgal/pull/306#discussion-diff-38633460

**Comment by lrineau** [comment]
Other TODO: [compilation error on Windows][e]. The function called here are called directory in the global namespace. It should call methods of the `viewer` object.

[e]: https://cgal.geometryfactory.com/CGAL/Members/testsuite/CGAL-4.7-Ic-134/Polyhedron_Demo/TestReport_afabri_x64_Cygwin-Windows8_MSVC2012-Release-32bits.gz

**Comment by maxGimeno** [comment]
I removed the semi-column

**Comment by lrineau** [comment]
The *semi-colon*. :-)

Thanks for the fix.

**Comment by lrineau** [comment]
There is still another error, from [CGAL-4.7-Ic-138 on Windows, with MSVC][I].

[I]: https://cgal.geometryfactory.com/CGAL/Members/testsuite/CGAL-4.7-Ic-138/Polyhedron_Demo/TestReport_afabri_x64_Cygwin-Windows8_MSVC2012-Release-32bits.gz
```
[...]\Scene_points_with_normal_item.cpp(410) : error C2039: 'glMultiTexCoord1d' : is not a member of '`global namespace''
[...]\Scene_points_with_normal_item.cpp(410) : error C3861: 'glMultiTexCoord1d': identifier not found
```

**Comment by lrineau** [comment]
And, in addition to the error in `Scene_points_with_normal_item.cpp`, there is also [a warning][w] in `Scene.cpp`:
```
Polyhedron_Demo/Scene.cpp:24:6: warning: 'void {anonymous}::CGALglcolor(QColor)' defined but not used [-Wunused-function]
```

[w]: https://cgal.geometryfactory.com/CGAL/Members/testsuite/CGAL-4.7-Ic-138/Polyhedron_Demo/TestReport_lrineau_x86-64_Linux-Fedora19_g++-4.8_Release.gz

**Pull request merged**